docs/training-apis/api-pytorch.txt

```diff
@@ -1,43 +1,141 @@
-###############################
- Python API determined.pytorch
-###############################
+#############
+ PyTorch API
+#############
 
-.. _pytorch-trial:
+**********
+ Overview
+**********
+
+This document lists how you should implement the PyTorch API. You need to implement a trial class
```

Comment:
This document ~lists how you should implement the PyTorch API~ guides you through training a PyTorch model in Determined, by implementing a PyTorchTrial.

Reason: "listing" is the wrong verb, because when you "list XYZ", the "XYZ" needs to be multiple things.  "how you should implement" is just one thing.

Also, "how you should implement" only makes sense for a plugin, but we also document the Context object, which a user does not have to implement at all.

Also, just after this, you describe the Plugin API itself, but the whole document is not about the plugin API itself, which makes this overview extra confusing.